### ISSUE TITLE
DSSP E and B characters do not match original Kabsch & Sander 1983 definitions

### ISSUE THREAD

**Issue by drroe** [Amber-MD/cpptraj#691]
From the original paper:
```
“B” = residue in isolated beta-bridge 
“E” - extended strand. participates in beta-ladder
```
However, cpptraj uses `E` to mean parallel beta and `B` to mean anti-parallel. May want to introduce a keyword that brings output in line with DSSP definition.

**Comment by saikat0003** [comment]
Dear Sir,
Is this issue solved?

**Comment by drroe** [comment]
Not yet - I've been swamped with other projects for a while but I'm working on it now. The DSSP algorithm needs a bit more reworking beyond this issue, so it's taking a little longer than I had originally thought.

**Comment by drroe** [comment]
The original email: http://archive.ambermd.org/201903/0181.html

**Comment by drroe** [comment]
Should now be addressed by #748. @saikat0003 please let me know if you see any problems.
